include/roaring/portability.h

```diff
@@ -27,11 +27,10 @@
 #include <malloc.h> // this should never be needed but there are some reports that it is needed.
 #endif
 
-#if __SIZEOF_LONG_LONG__ != 8
+#if defined(__SIZEOF_LONG_LONG__) && __SIZEOF_LONG_LONG__ != 8
```

Comment:
@ksajme  Can you do it as part of your PR? I'd imagine something like

```
#if defined (_WIN32) && !defined(_WIN64)
#pragma message("You appear to beattempting a 32-bit build under Visual Studio. We recommend a 64-bit build instead.")
#endif 
```

General idea is to make a best-effort attempt at helping people.

Thoughts?